GoogleForCreators/web-stories-wp
# Optimize Image element loading

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes

When you click on the image in the media library you should instantly see the image on the canvas.

## Testing Instructions

When you click on the image in the media library you should instantly see the image on the canvas. It will use thumbnail so looking closely you will see low quality of the image, after a while (depending on your connection speed, DevTools connection limiter can be useful) you will see a better version of the image.

When image caching is disabled on your WordPress instance then you will see no instant effect.
Like on our QA env? Please check it on a local/clean WordPress instance.

---



Fixes #2615
